src/python/ensembl/io/genomio/utils/json_utils.py

```diff
@@ -30,7 +29,11 @@ def get_json(src_path: PathLike, **kwargs) -> Any:
 
     """
     with Path(src_path).open("r") as json_file:
-        return json.load(json_file, **kwargs)
+        try:
+            return json.load(json_file, **kwargs)
+        # Empty JSON file
+        except:
+            return {}
```

Comment:
Do we really want to return an empty dictionary when the JSON file cannot be read? As a general approach, I believe raising the exception is a better option, and leaving the `try`-`except` for the function calling `get_json()`.

Reply:
I'm afraid I would need to know more to be able to reply exactly. But if we are talking about cases like [`genome_stats/compare.py`](https://github.com/Ensembl/ensembl-genomio/blob/hackathon_nov23/src/python/ensembl/io/genomio/genome_stats/compare.py#L194-L197) where the code is currently only handling the file existing but being empty, I would then add `FileNotFoundError` to avoid stopping the pipeline, i.e. `except (FileNotFoundError, KeyError):`.

Reply:
In the case of an empty file, I would add `json.decoder.JSONDecodeError` to the `except` block instead.

Reply:
I've amended the Raise Error as suggested, revery json utils and ran black on code changes. Tested also in full nxflow pipeline run.